modules/gdscript/gd_parser.cpp

```diff
@@ -3094,9 +3089,9 @@ void GDParser::_parse_class(ClassNode *p_class) {
 							tokenizer->advance(); //var before the identifier is allowed
 						}
 
-						if (tokenizer->get_token() != GDTokenizer::TK_IDENTIFIER) {
+						if (!tokenizer->is_token_literal(0, true)) {
 
-							_set_error("Expected identifier for argument.");
+							_set_error("Expected literal for argument.");
```

Comment:
literal? It's still an identifier. It's just that the identifier might have the same name as a keyword, it's an identifier nonetheless.

Usually "constant values" (like a string or a number) are called literals. I'd keep the old error message.